docs/brand.md

```
@@ -0,0 +1,168 @@
+|---|---|---|---|
+| `primary` | `#0A7D89` | links, primary buttons, active states | 4.65:1 on ground, 4.88:1 on surface |
+| `primary-hover` | `#08646E` | hover of the above | 6.87:1 on surface |
+| `primary-text` | `#12212B` | body copy, headings | 15.66:1 |
+| `primary-text-muted` | `#4F636B` | secondary copy, labels' helper text | 6.01:1 |
+| `primary-text-inverted` | `#FFFFFF` | text on `primary` | 4.88:1 |
+| `primary-bg` | `#F7FAFB` | page ground | — |
+| `secondary-bg` | `#FFFFFF` | cards, inputs, alternating bands | — |
+| `accent-bg` | `#DFE7EA` | hairline borders, subtle fills, skeletons | — |
+| `border-strong` | `#768A91` | form-control edges | 3.61:1 (UI boundary ≥ 3:1) |
+| `accent` | `#1E40AF` | reserved for informational emphasis | 8.72:1 |
+| `secondary` | `#FB9999` | the map's active pin and nothing else — it fails as text (2.08:1) |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)